### PR TITLE
JHA: FPC distribution bar refactor.

### PR DESCRIPTION
**Pull request by harrymaynard-maximus**
### Please check if the PR fulfills these requirements:

- [x] After these changes, the app was run and still works as expected
- [ ] Unit tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass

### CODE REVIEW

File: jha/src/components/enrolment/ReviewTableList.vue [hunk 1]
```diff
@@ -142,16 +173,62 @@ export default {
...
+
+    try {
+      if (!apiData) {
+        const response = await apiService.getDeductibles(
+          this.$store.state.enrolmentModule.captchaToken,
+          this.$store.state.enrolmentModule.fpcUuid
+        );
+        apiData = response.data;
+        if (!apiData || !apiData.assistanceLevels || !apiData.pre1939AssistanceLevels) {
+          throw new Error('response data does not include assistance levels.');
```
Comment (bryce-mcmath) [inline jha/src/components/enrolment/ReviewTableList.vue:203]: Should there be some Splunk logging here?

**Review by stephanieharrison-maximus — COMMENTED** [review]
I don't know this component terribly well, but I see no problems.

**Review by stephanieharrison-maximus — APPROVED** [review]

File: jha/src/components/enrolment/ReviewTableList.vue (hunk 1 above)
Comment (harrymaynard-maximus) [reply jha/src/components/enrolment/ReviewTableList.vue:204]: Done. Added Splunk logging in the `catch` block.

**Review by bryce-mcmath — APPROVED** [review]
Nice

**Pull request merged**